src/map/party.cpp

```diff
@@ -60,29 +60,30 @@ struct CParty::partyInfo_t
     uint16      prev_zone;
 };
 
-/************************************************************************
- *                                                                      *
- *  Конструктор                                                         *
- *                                                                      *
- ************************************************************************/
-
+// Constructor
 CParty::CParty(CBattleEntity* PEntity)
 {
-    XI_DEBUG_BREAK_IF(PEntity == nullptr);
-    XI_DEBUG_BREAK_IF(PEntity->PParty != nullptr);
-
-    m_PartyID     = PEntity->id;
-    m_PartyType   = PEntity->objtype == TYPE_PC ? PARTY_PCS : PARTY_MOBS;
-    m_PartyNumber = 0;
+    m_PLeader        = nullptr;
+    m_PAlliance      = nullptr;
+    m_PSyncTarget    = nullptr;
+    m_PQuaterMaster  = nullptr;
+    m_EffectsChanged = false;
+    m_PartyID        = 0;
+    m_PartyType      = PARTY_MOBS;
+    m_PartyNumber    = 0;
 
-    m_PLeader       = nullptr;
-    m_PAlliance     = nullptr;
-    m_PSyncTarget   = nullptr;
-    m_PQuaterMaster = nullptr;
+    if (PEntity != nullptr && PEntity->PParty != nullptr)
```

Comment:
`if (PEntity != nullptr && PEntity->PParty != nullptr)` is throwing the CI warnings, `PEntity->PParty` should be null until it's assigned in `AddMember(PEntity);`, a second construction of PParty adding a member already in a party will throw the warning, needs an accompanying edit to the `ShowWarning`

I guess some mobs are in parties for some reason which is calling this constructor on startup which is good for this PR